Bottom: 0b5f3e471409384ccdec01309ac99bdd4ac8444a
Top:    103c1733f9bb7cf3b99a177d3a6541b78b25395b
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 16:40:02 -0500

Refresh of all-docker_volume

---
diff --git a/roles/docker-exim4/vars/main.yml b/roles/docker-exim4/vars/main.yml
index 20d48fc..0e72776 100644
--- a/roles/docker-exim4/vars/main.yml
+++ b/roles/docker-exim4/vars/main.yml
@@ -9,6 +9,9 @@ exim4_image_name: '{{ docker_registry_username }}/exim4'
 # the name of the container being started
 exim4_container_name: '{{ docker_name_prefix }}exim4'
 
+# the name of the config-volume used by the container
+exim4_cv_name: '{{ docker_name_prefix }}exim4_CV'
+
 # the name of the data-volume used by the container
 exim4_dv_name: '{{ docker_name_prefix }}exim4_DV'
